suite-common/wallet-types/src/transaction.ts

```diff
@@ -139,7 +139,15 @@ export type PrecomposedTransactionCardano =
     | PrecomposedTransactionCardanoNonFinal
     | PrecomposedTransactionCardanoFinal;
 
-export type GeneralPrecomposedTransaction = PrecomposedTransaction | PrecomposedTransactionCardano;
+export type PrecomposedTransactionStellar = PrecomposedTransaction & {
```

Comment:
Please avoid special types for Stellar. We also plan to get rid of `PrecomposedTransactionCardano`

Reply:
https://github.com/trezor/trezor-suite/pull/16483/files#diff-99a8863ba768d5bd3a372c152bfc30d6ea7263ac643260db82ccb92571236d0dL289-L294